src/avro/src/reader.rs

```diff
@@ -329,6 +329,7 @@ pub struct SchemaResolver<'a> {
     pub writer_to_reader_names: BTreeMap<usize, usize>,
     pub reader_to_writer_names: BTreeMap<usize, usize>,
     pub reader_to_resolved_names: BTreeMap<usize, usize>,
+    #[allow(dead_code)]
     pub reader_fullnames: BTreeMap<usize, &'a FullName>,
```

Comment:
I marked this as `dead_code` instead of removing it because I don't know anything about the avro schema resolver and if this field might be generally useful. More than happy to remove it if folks would prefer that